customer/client.go

```diff
@@ -31,9 +31,11 @@ func (c Client) New(params *stripe.CustomerParams) (*stripe.Customer, error) {
 		}
 
 		if len(params.Token) > 0 {
-			body.Add("card", params.Token)
+			body.Add("source", params.Token)
 		} else if params.Card != nil {
 			params.Card.AppendDetails(body, true)
```

Comment:
if I understand what you're asking: it doesn't have to --  the backend is set up to take `card` or `source` for backwards compatibility so users can continue to use Card if they're keen on it